src/routes/new-reservation/+page.svelte

```diff
@@ -0,0 +1,448 @@
+                                                    <th> Marke: </th>
+                                                    <td>
+                                                        {car.brand}
+                                                    </td>
+                                                </tr>
+                                                <tr>
+                                                    <th> Typ: </th>
+                                                    <td>
+                                                        {car.type}
+                                                    </td>
+                                                </tr>
+                                                <tr>
+                                                    <th> Leistung: </th>
+                                                    <td>
+                                                        {car.ps} PS
+                                                    </td>
+                                                </tr>
+                                                <tr>
+                                                    <th> Sitzplätze: </th>
+                                                    <td>
+                                                        {car.seats}
+                                                    </td>
+                                                </tr>
+                                            </table>
+                                        </div>
+                                        <div
+                                            class="col-span-1 flex h-full flex-col items-center justify-center"
+                                        >
+                                            <input
+                                                class="radio align-middle"
+                                                type="radio"
+                                                name="radio-button"
+                                                value={index}
+                                                id="radio_{index}"
+                                            />
+                                        </div>
+                                    </div>
+                                </a>
+                            {/each}
+                        </div>
+
+                        <!-- Navigation -->
+                        <div class="grid grid-cols-2 space-x-2">
+                            <button
+                                class="variant-filled-primary btn"
+                                on:click={() => {
+                                    formPage = 0;
+                                }}>Zurück</button
+                            >
+                            <button
+                                class="variant-filled-primary btn"
+                                on:click={() => {
+                                    if (checkAndSaveCarSelected(loadedCars)) {
```

Comment:
Hier dann den Methodennamen von oben übernehmen
```suggestion
                                    if (checkCarSelected()) {
```